react/src/components/SessionDetailContent.tsx

```diff
@@ -347,8 +351,41 @@ const SessionDetailContent: React.FC<{
               />
             </Descriptions.Item>
           ) : null}
-          <Descriptions.Item label={'Resource Usage'} span={md ? 2 : 1}>
-            <SessionUsageMonitor sessionFrgmt={session} />
+          <Descriptions.Item
+            label={
+              <Flex direction="column" align="start" gap={token.marginSM}>
+                <Typography.Text style={{ color: token.colorTextSecondary }}>
+                  {t('session.ResourceUsage')}
+                </Typography.Text>
+                <Select
+                  size="small"
```

Comment:
```suggestion
                  size="small"
                  popupMatchSelectWidth={false}
                  variant="filled"
```
To resolve below issue, please set `popupMatchSelectWidth` to false.
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/2HueYSdFvL8pOB5mgrUQ/c62e351e-dfb0-412e-b08a-7c51ac4fbb2c.png)
---
To Be
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/2HueYSdFvL8pOB5mgrUQ/72554298-072c-47c0-9306-875ca67ad3b1.png)